scenario_player/main.py

```diff
@@ -75,7 +75,7 @@ def main(ctx, scenario_file, keystore_file, password, chains, data_path, auth, m
     else:
         scenario_basename = basename(scenario_file.name)
         log_file_name = (
-            f"scenario-player_{scenario_basename}_{datetime.now():%Y-%m-%dT%H:%M:%S}.log"
+            f"{os.environ['HOME']}/.raiden/scenario-player/scenarios/{scenario_basename}/{scenario_basename}_{datetime.now():%Y-%m-%dT%H:%M:%S}.log"
```

Comment:
This should re-use the `data_path` that is passed in from the command line:

https://github.com/raiden-network/scenario-player/blob/a70c56970b8ea311cdec2ca3f757bcc6aaab132e/scenario_player/main.py#L115

https://github.com/raiden-network/scenario-player/blob/a70c56970b8ea311cdec2ca3f757bcc6aaab132e/scenario_player/runner.py#L66

Also it would make sense to include the run number in the log name (as it's done for the node logs).

Reply:
huh. I missed that variable. will do!

Reply:
As for the run number, I think that's determined somewhere deeper in the code, so I'm leaving it our for now.

Reply:
Yeah that's true. Since the debug log file uses the `RotatingFileHandler` we should get separate files for every run anyway.